frontend/.env.example

```
@@ -0,0 +1,2 @@
+VITE_SMART_CONTRACT=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)